QAutoLibrary/QAutoSelenium.py

```diff
@@ -210,6 +255,53 @@ def measure_async_screen_response_time(self, measurement_name, timeout, referenc
 
         return False
 
+    def get_last_element_details(self):
+            x = self.last_element.coordinates[0]
+            y = self.last_element.coordinates[1]
+            w = self.last_element.size[0]
+            h = self.last_element.size[1]
+            driver = self.driver_cache._get_current_driver()
+            dimensions = driver.get_window_size()
+            
+            print("EXPECTED ELEMENT: " + '(' + "By." +  self.last_element[0].upper() + ", " + self.last_element[1] + ')')
+            print("Element location x: " + str(x) + " y: " + str(y) + " width: " + str(w) + " height: " + str(h))
+            print("Screen dimensions width: " + str(dimensions.get("width")) + " height: " + str(dimensions.get("height")))
+
+            # check if coordinates are empty or outside current screen
+            if str(x) != "" and str(y) != "" and str(w) != "" and str(h) != "" and dimensions.get("height") > int(x) and dimensions.get("width") > int(y):
+                width = int(w) / 2
+                height = int(h) / 2
+                new_x = int(x) + width
+                new_y = int(y) + height
+                elem_str = "return document.elementFromPoint(" + str(new_x) + ", " + str(
+                    new_y) + ")"
+                element = driver.execute_script(elem_str)
+                
+                xpath = ""
+                try:
+                    current = element
+                    path = ''
+                    while current.tag_name.lower() != "html":
+                        current_path = driver.execute_script(JS_GET_XPATH_POSITION, current)
+                        path = current_path + path
+                        xpath = '/' + path
+                        elements = driver.find_elements_by_xpath(xpath)
+                        if len(elements) == 1 and element == elements[0]:
+                            break
+                        current = driver.execute_script("return arguments[0].parentNode", current)
+                           
+                    xpath = xpath
+                except WebDriverException as e:
+                    print(e)
+                    pass
+                
+                print("ACTUAL ELEMENT DETAILS:")        
+                print("Element id: " + str(element.get_attribute('id')))
+                print("Element class name: " + str(element.get_attribute('class')))
+                print("Element xpath: " + str(xpath))
+                print("Element tag name: " + str(element.tag_name))
+                print("Element text: " + str(element.text))
```

Comment:
Is build in str function needed here at all? I thing that at least str(element.text) is not working if there is non ascii characters in element i.e "äö"